pvlib/irradiance.py

```diff
@@ -323,38 +323,50 @@ def get_total_irradiance(surface_tilt, surface_azimuth,
     Parameters
     ----------
     surface_tilt : numeric
-        Panel tilt from horizontal.
+        Panel tilt from horizontal.[degree]
     surface_azimuth : numeric
-        Panel azimuth from north.
+        Panel azimuth from north. [degree]
     solar_zenith : numeric
-        Solar zenith angle.
+        Solar zenith angle. [degree]
     solar_azimuth : numeric
-        Solar azimuth angle.
+        Solar azimuth angle. [degree]
     dni : numeric
-        Direct Normal Irradiance
+        Direct Normal Irradiance. [W/m2]
     ghi : numeric
-        Global horizontal irradiance
+        Global horizontal irradiance. [W/m2]
     dhi : numeric
-        Diffuse horizontal irradiance
+        Diffuse horizontal irradiance. [W/m2]
     dni_extra : None or numeric, default None
-        Extraterrestrial direct normal irradiance
+        Extraterrestrial direct normal irradiance. [W/m2]
     airmass : None or numeric, default None
-        Airmass
+        Relative airmass (not adjusted for pressure). [unitless]
     albedo : numeric, default 0.25
-        Surface albedo
+        Surface albedo. [unitless]
     surface_type : None or String, default None
-        Surface type. See grounddiffuse.
+        Surface type. See :py:func:`~pvlib.irradiance.grounddiffuse` for
+        the list of accepted values.
     model : String, default 'isotropic'
-        Irradiance model.
+        Irradiance model. Can be one of 'isotropic', 'klucher', 'haydavies',
+        'reindl', 'king', 'perez'.
     model_perez : String, default 'allsitescomposite1990'
-        Used only if model='perez'. See :py:func:`perez`.
+        Used only if model='perez'. See :py:func:`~pvlib.irradiance.perez`.
 
     Returns
     -------
     total_irrad : OrderedDict or DataFrame
         Contains keys/columns ``'poa_global', 'poa_direct', 'poa_diffuse',
         'poa_sky_diffuse', 'poa_ground_diffuse'``.
+
```

Comment:
```suggestion

    Raises
    ------
    ValueError
        If model is one of ``'haydavies'``, ``'reindl'``, or ``'perez'`` and ``dni_extra``
        is ``None``.

```

Reply:
I agree that the `ValueError` is raised, but it is raised in a function that is called by `get_total_irradiance`. So include the `Raises` block here also?

Reply:
IDK. @wholmgren ? I think the user doesn't actually know that it calls `get_diffuse_sky()` so either:
* alerting them to why it may raise `ValueError` is polite and not redundant
* or add somewhere that this calls `get_diffuse_sky()` possible in a _See Also_ section?
your call. I will not choose the color of the _white_ bike shed 😉

Reply:
In my review this morning I started typing an opinion on this and then deleted it because I changed my mind twice. Not going near that bike shed with a 10 foot roller. Probably no wrong answer here.

Reply:
There's paint on the shed - good enough for me.

A future maintainer may wonder why the docstring has a `Raises` clause when the method contains no `raise` statement. That's my objection.